src/main/java/com/example/bookstoreapp/filters/AuthenticationFilter.java

```diff
@@ -19,62 +19,62 @@
 @Slf4j
 public class AuthenticationFilter implements Filter {
 
-  private static final String[] PUBLIC_URI_PATTERNS = {"/api/**", "/user/login", "/user/signup", "/health"};
+    private static final String[] PUBLIC_URI_PATTERNS = {"/api/**", "/user/login", "/user/signup", "/health"};
```

Comment:
You should not commit filter changes here because your implementation does not need filter to be modified. Remove this change from your pull request l.